matrix-neoboard-widget/vite.config.ts

```diff
@@ -28,6 +28,8 @@ if (process.env.VITE_DEV_SSL === 'true') {
 
 // https://vitejs.dev/config/
 export default defineConfig({
+  // Default is "/" what would make the widget only work when not deployed in a sub-path
```

Comment:
sentence is confusing :D feels like a trivial change but either we lose the comment or make it clearer.